platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/XMLUtilsTest.java

```diff
@@ -100,6 +102,12 @@ public void testTransformNode() throws ParserConfigurationException, IOException
         assertThat(result.getWriter().toString(), is(equalTo(XML)));
     }
 
+    @Test
+    public void testGetRootNamespace() {
+        assert "doggy-namespace".equals(XMLUtils.getRootNamespace(XML_WITH_NAMESPACE));
```

Comment:
Should use assertThat(XMLUtils.getRootNamespace(XML_WITH_NAMESPACE), is(equalTo("doggy-namespace"));